promscale/page-index/page-index.js

```diff
@@ -49,6 +49,13 @@ module.exports = [
             keywords: ['Promscale', 'analytics'],
             excerpt: 'Install Promscale on bare metal',
           },
+          {
+            title: 'Resource recommendation guide',
+            href: 'recomm-guide',
+            tags: ['promscale', 'install', 'analytics'],
+            keywords: ['Promscale', 'analytics'],
+            excerpt: 'Resource recommendation guide',
```

Comment:
```suggestion
            excerpt: 'Recommended resources for Promscale and TimescaleDB',
```

Reply:
I will add this as `Resource resources for Promscale` 

The definition of Promscale as a product is **Promscale = Promscale connector + TimescaleDB**.

Reply:
@VineethReddy02  Thanks!!! I just updated the excerpt